lib/contracts/relationship-flowdock-message-is-sent-by-flowdock-user.ts

```diff
@@ -0,0 +1,19 @@
+import type { RelationshipContractDefinition } from 'autumndb';
```

Comment:
On second thought, we should be able to use the wildcard `is-creator-of` relationship we currently use for other things in the system: https://github.com/product-os/jellyfish-worker/blob/master/lib/contracts/relationship-any-is-creator-of-any.ts

Meaning that we don't need this new relationship, apologies for that.

Reply:
Cool, how does that work. how do you specify the relation here?

Reply:
@iamsolankiamit We don't need to specify anything here. We just need to make sure we link flowdock user contracts and their messages with the `is-creator-of` link.

Reply:
aha, great!